### DISCUSSION TITLE
Cant figure out why my model is not accurate

### DISCUSSION THREAD

**Discussion by hmmonotone** [explosion/spaCy#12277] [Help: Model Advice]
I have tried to make a NER model with 2500 text and annotation data. But after training the model, it is giving wrong answers even for the test data. I can't figure out why. Need help. The data is very accurate as it is checked 5 times by analysts.

**Comment by kadarakos** [comment]
Hey,

Sounds disappointing indeed! What do you mean exactly by "even for the test data"? We usually talk about *training data* which is what the model sees, *development data* which is the one `spacy` uses to monitor training and *test data* which the model does not have access to and is only used for evaluation after development.  By "test data" do you mean what I call here "training data"?

**Reply by hmmonotone** [reply]
yes

**Comment by kadarakos** [comment]
Thanks for the clarification! Alright so let's see my next question would be to understand quantitatively what you mean by "it is giving wrong answers". Do you use the [`train`](https://spacy.io/api/cli#train) command to train the model and then [`benchmark`](https://spacy.io/api/cli#benchmark-accuracy) to get a single number to measure performance? If so, could you share output of `benchmark accuracy`?

**Reply by hmmonotone** [reply]
can you connect with me over email?

**Reply by hmmonotone** [reply]
hmishra.gaya@gmail.com

**Comment by kadarakos** [comment]
Please understand that we are not be able to provide individual support via email. We also believe that help is much more valuable if it's shared publicly, so that more people can benefit from it.

**Reply by hmmonotone** [reply]
Yes definitely. I will share the details in a while
